templates/data/generic_component_api_gateway_data.yaml

```diff
@@ -2,6 +2,11 @@ templates:
   - from: generic.tmpl
     render:
       - to: ../../prow/jobs/api-gateway/api-gateway-build.yaml
+        localSets:
+          job_spec:
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+            env:
+              GO111MODULE: "on"
```

Comment:
```suggestion

```

Reply:
`2022/09/06 10:37:40 Cannot parse data file templates/data/generic_component_api_gateway_data.yaml: yaml: line 47: found character that cannot start any token`